source/ocsp-support/ocsp-support.rst

```diff
@@ -437,11 +467,11 @@ Backwards Compatibility
 
 An application behind a firewall with an outbound whitelist that
 upgrades to a driver implementing this specification may experience
-connectivity issues. This is because the driver may need to contact
+connectivity issues when OCSP is enabled. This is because the driver may need to contact
```

Comment:
Clarifying that these backwards breaking changes should only apply when OCSP is enabled. Previously, we were defaulting to enabling OCSP even if drivers hard-failed. Now, this  applies only if an application opts in to OCSP and a driver exhibits hard-fail behavior.